src/genesis.lisp

```diff
@@ -1,29 +1,37 @@
 (in-package :emotiq/config)
 
 (defun genesis/create (configuration &key
-                                       root
+                                       (directory (emotiq/fs:tmp/))
                                        (force nil))
-  (let ((genesis-block-path (merge-pathnames
-                             (alexandria:assoc-value configuration
-                                                     :genesis-block-file)
-                             root)))
-    (when (or force
-              (not (probe-file genesis-block-path)))
-      (let ((genesis-block
-             (cosi/proofs:create-genesis-block
-              (alexandria:assoc-value configuration :address-for-coins)
-              (get-stakes))))
-        (with-open-file (o genesis-block-path
-                           :direction :output
-                           :if-exists :supersede)
-          (cl-json:encode-json genesis-block o))
+  (let ((create-genesis-block (uiop:find-symbol* :create-genesis-block :cosi/proofs)))
```

Comment:
The package may not exist. Check that first. Could wrap what's there with ```(and (find-package :cosi/proofs) ...)```.

Reply:
Grumble.  I thought that `uiop:find-symbol*` returned nil if the package didn't exist, but it doesn't.  Will fix.